caluma/caluma_core/urls.py

```diff
@@ -1,8 +1,14 @@
 from django.conf import settings
-from django.conf.urls import url
+from django.urls import path
 
+from caluma.caluma_core import views
 from caluma.caluma_user.views import AuthenticationGraphQLView
 
 urlpatterns = [
-    url(r"", AuthenticationGraphQLView.as_view(graphiql=settings.DEBUG), name="graphql")
+    path("healthz/", views.health_check_status, name="healthz"),
+    path(
+        "graphql/",
```

Comment:
Trailing slash should be optional:

```suggestion
    re_path("healthz/?", views.health_check_status, name="healthz"),
    re_path(
        "graphql/?",
```

Don't forget to import `re_path` instead of `path`.